go/git/get.go

```diff
@@ -230,7 +230,9 @@ func getMetadataInnerSingle(ctx context.Context, g *libkb.GlobalContext,
 	}
 
 	// Load UPAKs to get the last writer username and device name.
-	lastWriterUPAK, _, err := g.GetUPAKLoader().LoadV2(libkb.NewLoadUserArg(g).WithUID(responseRepo.LastModifyingUID))
+	lastWriterUPAK, _, err := g.GetUPAKLoader().LoadV2(libkb.NewLoadUserArgWithContext(ctx, g).
+		WithUID(responseRepo.LastModifyingUID).
+		WithPublicKeyOptional())
```

Comment:
I think that this missing `PublicKeyOptional` was leading to the `No public key found` error. Trap argument